### PR TITLE
Add table search for all tables in content-tagger

### PR DESCRIPTION
**Pull request by Davidslv**
Trello: https://trello.com/c/AP70lKei/94-add-table-search-for-all-tables-in-content-tagger

## Look & Feel

screen shot 2016-08-12 at 15 08 27

screen shot 2016-08-12 at 14 29 17
screen shot 2016-08-12 at 15 05 11
screen shot 2016-08-12 at 15 07 17

### CODE REVIEW

**Comment by carvil** [comment]
Nice one @Davidslv !

I think there is only one table missing [here](https://github.com/alphagov/content-tagger/blob/master/app/views/tagging_spreadsheets/_import_preview.html.erb#L10). In order to see it you will have to import a spreadsheet first. Please have a look into `content-tagger` in integration for an example spreadsheet to use locally. After importing it, click on `Preview` to see the table of `tag_mapping` associations.

On your last screenshot, do you think the search row could be stretched to use the entire width of the table?

File: app/views/tagging_spreadsheets/index.html.erb [hunk 1]
```diff
@@ -12,6 +12,7 @@
       <th></th>
       <th></th>
     </tr>
+    <%= render partial: 'shared/table_filter' %>
```
Comment (jackscotti) [inline app/views/tagging_spreadsheets/index.html.erb]: Shouldn't `'shared/table_filter'` be 'shared/_table_filter'?

Comment (tijmenb) [inline app/views/tagging_spreadsheets/index.html.erb]: This is the correct way to call partials: http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials

**Comment by Davidslv** [comment]
I've updated the PR, thanks @carvil

**Pull request merged**